frontend/console/src/api/timeline/use-request-trace-events.ts

```diff
@@ -7,7 +7,7 @@ export type TraceEvent = CallEvent | IngressEvent
 export const useRequestTraceEvents = (requestKey?: string, filters: EventsQuery_Filter[] = []) => {
   const eventTypes = [EventType.CALL, EventType.INGRESS]
   const allFilters = [...filters, requestKeysFilter([requestKey || '']), eventTypesFilter(eventTypes)]
-  const timelineQuery = useTimeline(true, allFilters, !!requestKey)
+  const timelineQuery = useTimeline(true, allFilters, 500, !!requestKey)
```

Comment:
Internally, there's currently a loop in `console.go` that is running at the interval we specify here. We might be able to avoid this, but it would probably be a new task to figure out how to trigger event refreshes when events are added.

Reply:
Got it. I'll create an issue to track that.

Reply:
https://github.com/TBD54566975/ftl/issues/2913